### ISSUE TITLE
Display sub-zones grouped together/hierarchical 

### ISSUE THREAD

**Issue by yanky83** [WillyXJ/facileManager#278]
It would be nice to have zones like

```
        example.com
    int.example.com
foo.int.example.com
```

to be displayed together or in some hierarchical  format.

**Comment by WillyXJ** [comment]
I looked into what it would take to support this and it will take a rewrite of basicGetList() to support this.  I may be able to include this in 3.0.

**Comment by yanky83** [comment]
I thought about it again, and maybe it would be enough to create an option to reverse the display of the domain name in the list, i.e.

com.example.int
com.example.int.dep

Maybe?

Cheers

**Comment by WillyXJ** [comment]
This is will be included in [3.0-alpha3](http://www.facilemanager.com/download/?t=4) and later.

**Comment by yanky83** [comment]
Yey :)

Good stuff!
